src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,17 @@
-export const CommentInfo = () => <>Put the comment here</>;
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment: { name, email, body } }) => (
```

Comment:
```suggestion
export const CommentInfo = ({ comment }) => {
  const { name, email, body } = comment;
```
It's not a good idea to do nested destructing, for better readability it's better to avoid it.